Scuti/Scripts/ScutiSDK/ScutiButton.cs

```diff
@@ -40,19 +42,58 @@ public void Start()
 
     public void OnClick()
     {
-        ScutiSDK.Instance.LoadUI();
-        NotificationIcon.SetActive(false);
+        if (canConnect)
+        {
+            ScutiSDK.Instance.LoadUI();
+            NotificationIcon.SetActive(false);
+        }
```

Comment:
Maybe call CheckNewOffers here if canConnect is false?

Reply:
Not good idea because it would need that the user press again to check, so in total the user would need to press two times